CHANGELOG.md

```diff
@@ -18,6 +14,14 @@
 
 ### Security
 
+## [0.4.0] - 2019-08-26
+
+### Added
+- Added a `intent_id` property when dispatching workflows and tasks, sending events to workflows, and
```

Comment:
This is a little bit messed up.
Blank lines should be after titles, and there should be no blank lines between items.

Reply:
Done